cpp/src/parquet/metadata.cc

```diff
@@ -1854,7 +1854,9 @@ class RowGroupMetaDataBuilder::RowGroupMetaDataBuilderImpl {
     row_group_->__set_file_offset(file_offset);
     row_group_->__set_total_compressed_size(total_compressed_size);
     row_group_->__set_total_byte_size(total_bytes_written);
-    row_group_->__set_ordinal(row_group_ordinal);
+    if (row_group_ordinal > 0) {
```

Comment:
By default, the `int16_t row_group_ordinal = -1`, but it's never set to that...

Reply:
Shouldn't this be
```suggestion
    if (row_group_ordinal >= 0) {
```